router/message.js

```diff
@@ -77,6 +77,9 @@ router.get('/authorName',(req,res) => {
     },
   );
 });
+router.get('/author', (req, res) => {
+  res.send('hi');
```

Comment:
make sure you update the response with proper data.

Reply:
ok.